src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $main-border-color;
+  background: $main-color;
+  transition: transform $transform-time;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &_photo {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    align-self: flex-start;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 24px;
+
+    &-grade {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $main-accent-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-tag {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-accent-color;
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    margin: auto;
+    width: 100%;
+    height: 40px;
+    border: 1px solid $moyo-accent-color;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    color: $main-color;
+    background-color: $moyo-accent-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: all $transform-time;
```

Comment:
Don't use `all` for transition, if affects in the bad way on the website performance
Specify only needed properties